src/RevisionPopupWidget.js

```diff
@@ -3,10 +3,23 @@ import Tools from './Tools';
 /**
  * @class
  * @constructor
+ * @param {Controller} controller WhoWroteThat controller
+ * @param {Model} model WhoWroteThat model
  */
-const RevisionPopupWidget = function RevisionPopupWidget() {
-	this.$popupContent = $( '<div>' )
+const RevisionPopupWidget = function RevisionPopupWidget( controller, model ) {
+	const $shimmerSpan = $( '<span>' )
+		.addClass( 'wwt-shimmer wwt-shimmer-animation' );
+
+	this.controller = controller;
```

Comment:
You're right; the first iteration, I had the widget itself invoke the fetch action when it toggled (which MVC dictated it needs to call the controller)... then, after a bit more thinking, I thought it might be easier to make the widget listen to the model which meant that the controller is invoked within the `app`

I have to admit, the widget listening to the model and the model sending a sort of "set token" event is okay in principle, but I'm not 100% in love with it. I do think it separates concerns much better, which is good, but maybe the model should actually retain the data/store it and do something with it rather than "just" be a conduit for the event.

... there was no reason for it for now, but it might be something to take into account. I'm not sure. No biggie, just pointing this out.

I'll remove `this.controller` from the widget since it does no actual action and only listens to state events from the model.